project/v1/types.go

```diff
@@ -108,4 +111,9 @@ type ProjectRequest struct {
 	DisplayName string `json:"displayName,omitempty" protobuf:"bytes,2,opt,name=displayName"`
 	// description is the description to apply to a project
 	Description string `json:"description,omitempty" protobuf:"bytes,3,opt,name=description"`
+	// udnName is the User Defined Networks Name to apply to a project
+	// Reference of regex and max length: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set
```

Comment:
Don't include a documentation reference here. Instead, state in plain sentences what the constraints of this field are.